vmdb/app/controllers/host_controller.rb

```diff
@@ -141,23 +136,48 @@ def show
     end
   end
 
+  def drift_history
```

Comment:
@bmclaughlin i think instead of adding all these new methods and set_config calls in some of the other methods, you can try to add the follwing, hopefully this will work:

def show_association(action, display_name, listicon, method, klass, association = nil)
super
set_config(identify_record(params[:id]))
end